tests/test_submit_tuxbuild.py

```diff
@@ -52,243 +63,88 @@ def submit_tuxbuild(self, tuxbuild):
         return proc
 
     def test_submit_tuxbuild_build(self):
-        proc = self.submit_tuxbuild("tests/data/submit/tuxbuild/build.json")
+        proc = self.submit_tuxbuild(os.path.join(self.build_dir, "build.json"))
         self.assertTrue(proc.ok, msg=proc.err)
-        self.assertTrue(proc.err.count("Submitting 1 tests, 1 metrics") == 3)
-        project = self.squad.group("my_group").project("my_project")
-
-        # Check results for next-20201021, which has 2 instances in build.json
-        build = project.build("next-20201021")
-
-        base_kconfig = [
-            'defconfig',
-            'https://raw.githubusercontent.com/Linaro/meta-lkft/sumo/recipes-kernel/linux/files/lkft.config',
-            'https://raw.githubusercontent.com/Linaro/meta-lkft/sumo/recipes-kernel/linux/files/lkft-crypto.config',
-            'https://raw.githubusercontent.com/Linaro/meta-lkft/sumo/recipes-kernel/linux/files/distro-overrides.config',
-            'https://raw.githubusercontent.com/Linaro/meta-lkft/sumo/recipes-kernel/linux/files/systemd.config',
-            'https://raw.githubusercontent.com/Linaro/meta-lkft/sumo/recipes-kernel/linux/files/virtio.config',
-        ]
+        self.assertTrue(proc.err.count("Submitting 1 tests, 1 metrics") == 1)
 
-        # Make sure metadata values match expected values
-        urls = ['https://builds.tuxbuild.com/%s/' % _id for _id in ['B3TECkH4_1X9yKoWOPIhew', 't8NSUfTBZiSPbBVaXLH7kw']]
-        configs = [url + "config" for url in urls]
-        expected_metadata = {
-            'git_repo': "https://gitlab.com/Linaro/lkft/mirrors/next/linux-next",
-            'git_ref': None,
-            'git_commit': "5302568121ba345f5c22528aefd72d775f25221e",
-            'git_sha': "5302568121ba345f5c22528aefd72d775f25221e",
-            'git_short_log': '5302568121ba ("Add linux-next specific files for 20201021")',
-            'git_describe': "next-20201021",
-            'kconfig': [base_kconfig + ["CONFIG_ARM64_MODULE_PLTS=y"], base_kconfig + ["CONFIG_IGB=y", "CONFIG_UNWINDER_FRAME_POINTER=y"]],
-            'git_branch': os.environ.get("KERNEL_BRANCH"),
-            'make_kernelversion': "5.9.0",
-            'kernel_version': "5.9.0",
-            'config': configs,
-            'download_url': urls,
-        }
-        for expected_key in expected_metadata.keys():
-            self.assertEqual(expected_metadata[expected_key], getattr(build.metadata, expected_key))
+        group = self.squad.group("my_group")
+        self.assertIsNotNone(group)
```

Comment:
`assertIsNotNone` isn't needed here, it's already checked when you use `self.submit_tuxbuild` because it requires group and project to exist.